Source/JavaScriptCore/yarr/YarrParser.h

```diff
@@ -51,31 +51,31 @@ enum class CharacterClassSetOp : uint8_t {
 template <class T> concept YarrSyntaxCheckable = requires (T& checker, Vector<Vector<char32_t>>& disjunctionStrings, const String& subpatternName) {
     { checker.assertionBOL() } -> std::same_as<void>;
     { checker.assertionEOL() } -> std::same_as<void>;
-    { checker.assertionWordBoundary(bool{}) } -> std::same_as<void>;
```

Comment:
This is all just cleaning up style, since we weren't consistent with our usage of whitespace.